mkdocs.yml

```diff
@@ -1,7 +1,7 @@
 site_name: Docs CSC
 edit_uri: "https://github.com/CSCfi/csc-user-guide/edit/master/docs"
 site_url: "https://docs.csc.fi/"
-site_description: Instructions and user guides for the CSC supercomputer Puhti, Allas storage, Pouta, Rahti, software list, FAQ and tutorials.
+site_description: Instructions and user guides for the CSC supercomputer Mahti, Puhti, Allas storage, Pouta, Rahti, software list, FAQ and tutorials.
```

Comment:
```suggestion
site_description: Instructions and user guides for the CSC supercomputers, cloud services, storage and software, including FAQ and tutorials
```

Reply:
I don't think that's a good idea. This is metadata and according to google analytics puhti is one of the main keywords leading to our site. Thus, we should tell google (via site metadata) that these pages are about puhti (and mahti). The metadata is not for human reading as far as I understand. The original list was suggested and approved by Haiyan.

Reply:
```suggestion
site_description: Instructions and user guides for the CSC supercomputers Mahti and Puhti, Allas storage, cloud services Pouta and Rahti, and software – including FAQ and tutorials
```